api/v1alpha1/eacruntime_types.go

```diff
@@ -154,7 +154,7 @@ type EACRuntimeSpec struct {
 
 	// CleanCachePolicy defines cleanCache Policy
 	// +optional
-	CleanCachePolicy CleanCachePolicy `json:"cleanCachePolicy,omitempty"`
+	CleanCachePolicy CleanCacheBehavior `json:"cleanCachePolicy,omitempty"`
```

Comment:
I've checked some other projects. Seems policy is widely used https://github.com/kubeflow/common/blob/master/pkg/apis/common/v1/types.go#L146 than behavior.